generator/docs/NullOperator.md

```diff
@@ -0,0 +1,168 @@
+# `null`, presence, absence and the empty set.
+
+The `null` operator in a profile, expressed as `"is": "null"` or the negated equivalent has several meanings. It can mean (and emit the behaviour) as described below:
+
+| Absence / Presence<br />Described in other technologies as nullability | Field value<br /> |
+| ---- | ---- |
+| _null operator omitted_<br /> **The default**. The field's value may be absent or present | `is null`<br />The field will have _no value_ |
+| `not(is null)`<br />The field's value must be present | `not(is null)`<br />The field must have a value |
+
```

Comment:
should all 3 options from the list below be present in this table? The top line seems to combine the absence of a specified null operator and the presence of an "is null" constraint

Reply:
Updated to reference each cell int the table from the list.